lib/BinLib.sol

```diff
@@ -278,7 +278,7 @@ library BinLib {
   }
 
   // see note posIn*
-  // note with int24 bin we only use 2 bits in root
+  // note with int20 bin we only use 2 bits in root
```

Comment:
Tick is 21 bits, so I guess bin must be the same and cannot fit in `int20`?

Reply:
fixed in 1a4f5ad2db572b5b7ca732e7585ab93c0bc6c5a7